tests/sentry/tasks/test_delete_seer_grouping_records.py

```diff
@@ -1,10 +1,9 @@
 from time import time
-from unittest.mock import patch
+from unittest.mock import MagicMock, patch
 
 from sentry.models.grouphash import GroupHash
 from sentry.tasks.delete_seer_grouping_records import (
     call_delete_seer_grouping_records_by_hash,
-    call_seer_delete_project_grouping_records,
```

Comment:
Later in this file I remove the test for this function since we have a better test here (since it's used in that endpoint):
https://github.com/getsentry/sentry/blob/c20ff4ce3e92b2b9380a0f6aaa6424f06c6fa388/tests/sentry/api/endpoints/test_project_details.py#L1555-L1563